src/dom/dom.js

```diff
@@ -2173,10 +2173,14 @@ p5.Element.prototype.remove = function() {
   // stop all audios/videos and detach all devices like microphone/camera etc
   // used as input/output for audios/videos.
   if (this instanceof p5.MediaElement) {
-    const tracks = this.elt.srcObject.getTracks();
-    tracks.forEach(function(track) {
-      track.stop();
-    });
+    this.stop();
+    let sources = this.elt.srcObject;
```

Comment:
I think `let` could be `const` on this line and the line two below here. otherwise looks good!

Reply:
@lmccart I have resolved the issue. Please have a look